pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -497,12 +500,51 @@ func validatingHasNoSideEffects(webhooks []admissionregistration.ValidatingWebho
 	return true
 }
 
+// validatingWebhookAllowInvalidLabelValueInSelector returns true if all webhooksallow invalid label value in selector
+func validatingWebhookAllowInvalidLabelValueInSelector(webhooks []admissionregistration.ValidatingWebhook) bool {
+	for _, hook := range webhooks {
+		if hook.NamespaceSelector != nil {
+			allErrors := metav1validation.ValidateLabelSelector(hook.NamespaceSelector, false, field.NewPath("namespaceSelector"))
+			if len(allErrors) > 0 {
```

Comment:
can simplify to checking length on the ValidateLabelSelector call and passing in nil for the fieldpath since we're not using the content of the returned errors (applies to all calls like this)

```suggestion
			if len(metav1validation.ValidateLabelSelector(hook.NamespaceSelector, false, nil)) > 0 {
```